test/UnitTests/MSTestAdapter.UnitTests/Discovery/AssemblyEnumeratorTests.cs

```diff
@@ -131,8 +131,9 @@ public void GetTypesShouldLogWarningsWhenReflectionFailsWithLoaderExceptions()
         mockAssembly.Setup(a => a.GetTypes()).Throws(new ReflectionTypeLoadException(null, exceptions));
         mockAssembly.Setup(a => a.GetTypes()).Throws(new ReflectionTypeLoadException(null, exceptions));
 
-        IReadOnlyList<Type> types = AssemblyEnumerator.GetTypes(mockAssembly.Object, "DummyAssembly", _warnings);
+        Type[] types = AssemblyEnumerator.GetTypes(mockAssembly.Object, "DummyAssembly", _warnings);
 
+        Verify(types is not null);
```

Comment:
This assert is failing. It's because the mocking two lines above is throwing `ReflectionTypeLoadException` passing null array.